kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/spring-purap.xml

```diff
@@ -220,4 +220,6 @@
 		</property>
     </bean>
 
+	<bean id="taxElectronicFilingService" class="edu.arizona.kfs.module.purap.document.service.impl.ElectronicFilingServiceImpl" />
```

Comment:
Fixed. In making the change to Spring Injection, it seems that 2 helper objects are unused. They have been removed.